test/deployframework/context.go

```diff
@@ -311,30 +311,10 @@ func (ctx *DeployerCtx) Upgrade(packageName, repoVersion string, purgeReports, p
 		}
 	}
 
-	err = CreateUpgradeConfigMap(ctx.Logger, packageName, ctx.Namespace, ctx.HackScriptPath)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create the %s ConfigMap: %v", packageName, err)
-	}
-
-	err = VerifyConfigMap(ctx.Logger, ctx.Client, packageName, ctx.Namespace)
-	if err != nil {
-		return nil, fmt.Errorf("failed to verify the %s ConfigMap was successfully created: %v", packageName, err)
-	}
-
-	err = CreateCatalogSource(ctx.Logger, packageName, ctx.Namespace, packageName, ctx.OLMV1Alpha1Client)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create the %s CatalogSource: %v", packageName, err)
-	}
-
-	err = VerifyCatalogSourcePod(ctx.Logger, ctx.Client, packageName, ctx.Namespace)
-	if err != nil {
-		return nil, fmt.Errorf("failed to verify the %s CatalogSource was successfully created: %v", packageName, err)
-	}
```

Comment:
We can delete all of this logic now as we don't need to spin up a CatalogSource that references a ConfigMap anymore and can instead reference an existing CatalogSource that uses the registry image provided to the e2e suite.